funding/manager.go

```diff
@@ -2579,6 +2593,28 @@ func (f *Manager) sendFundingLocked(
 	return nil
 }
 
+// receivedFundingLocked checks whether or not we've received a FundingLocked
+// from the remote peer. If we have, RemoteNextRevocation will be set.
+func (f *Manager) receivedFundingLocked(chanID lnwire.ChannelID) (bool,
+	error) {
+
+	// If the funding manager has exited, return an error to stop looping.
+	select {
+	case <-f.quit:
+		return false, ErrFundingManagerShuttingDown
+	default:
+	}
+
+	channel, err := f.cfg.FindChannel(chanID)
```

Comment:
Re comment above: I think we can repurpose this field to allow a notification to be sent/received here: https://github.com/lightningnetwork/lnd/blob/master/funding/manager.go#L527-L528

There's also an interaction with another goroutine that we'll need to account for: https://github.com/lightningnetwork/lnd/blob/master/funding/manager.go#L2830-L2834

Reply:
that's true, but i wanted to make sure the state machine was consistent. the approach i used makes it so that the edge doesn't exist in the graph and another subsystem won't try to send a channel update if we've not received the funding locked -- it just essentially stops the flow